intellij/org.eclipse.xtext.idea/src/org/eclipse/xtext/idea/trace/VirtualFileBasedTrace.xtend

```diff
@@ -178,7 +184,8 @@ class VirtualFileBasedTrace extends AbstractTrace implements IIdeaTrace {
 	}
 
 	override IdeaModuleConfig getLocalProjectConfig() {
-		return new IdeaModuleConfig(getLocalProject());
+		val module = localProject
+		return workspaceConfigProvider.getWorkspaceConfig(module).findProjectByName(module.name)
```

Comment:
Could we IdeaModuleConfig as a localProject for VirtualFileBasedTrace? Only client now is org.eclipse.xtext.idea.trace.TraceForVirtualFileProvider.newAbstractTrace(VirtualFileInProject) which actually already has an instance of IdeaModuleConfig.